front/app/component-library/components/CollapsibleContainer/index.tsx

```diff
@@ -100,6 +87,22 @@ const CollapseContainer = styled(Box)`
   }
 `;
 
+type CollapsibleContainerTitleProps = {
+  titleVariant: TitleProps['variant'];
```

Comment:
I'm using the prop types of the prop destination (`TitleProps`) so we don't need to redefine them.